make/autoconf/lib-tests.m4

```diff
@@ -308,6 +308,32 @@ AC_DEFUN_ONCE([LIB_TESTS_SETUP_JIB],
   AC_SUBST(JIB_HOME)
 ])
 
+# Setup the tidy html checker
+AC_DEFUN_ONCE([LIB_TESTS_SETUP_TIDY],
+[
+  UTIL_LOOKUP_PROGS(TIDY, tidy)
+
+  if test "x$TIDY" != x; then
+    AC_MSG_CHECKING([if tidy is working properly])
+    tidy_output=`$TIDY --version 2>&1`
+    if ! $ECHO "$tidy_output" | $GREP -q "HTML Tidy" 2>&1 > /dev/null; then
+      AC_MSG_RESULT([no])
+      AC_MSG_NOTICE([$TIDY is not a valid tidy executable and will be ignored. Output from --version: $tidy_output])
+      TIDY=
+    elif ! $ECHO "$tidy_output" | $GREP -q "version" 2>&1 > /dev/null; then
+      AC_MSG_RESULT([no])
+      AC_MSG_NOTICE([$TIDY is missing a proper version number and will be ignored. Output from --version: $tidy_output])
+      TIDY=
+    else
+      AC_MSG_RESULT([yes])
+      AC_MSG_CHECKING([for tidy version])
+      TIDY_VERSION=`$ECHO $tidy_output | $SED -e 's/.*version //g'`
```

Comment:
Looks like `TIDY_VERSION` isn't "exported", should it be lower-case?

Reply:
We're not really consistant on that usage. In fact, I'd say that most variables are upper case, regardless of how "local" their use is, and the lower case variant is a minority. So I'd say it's up to Nizar; if he wants to change this he can, otherwise it can stay upper case.

Reply:
I lowercased it just to be safe.